.github/workflows/on-pullrequest.yml

```diff
@@ -88,10 +88,28 @@ jobs:
       - name: Build
         run: ./build.ps1 Build -Configuration Debug
 
-      - name: Run Unit Tests
+      - name: Install Coverlet Reference
         if: success()
+        run: |
+          dotnet tool install --global coverlet.console
+          dotnet tool install --global dotnet-reportgenerator-globaltool
+
+      - name: Run Unit Tests
         run: ./build.ps1 UnitTest -Configuration Debug
 
+      - name: Generate Coverage Report
+        if: always()
```

Comment:
What will happen if we run `reportgenerator` and there is no coverage file, because line 98 failed?